src/js/featureprompt.js

```diff
@@ -0,0 +1,26 @@
+import { get } from 'svelte/store';
+import { FeaturePrompt } from './constants.js';
+import { displayedMessages } from './sources-aggregate.js';
+import { index } from './sources-util.js';
+import { promptToShow } from './store.js';
+
+export const enable = () => {
+  const pushedPrompts = new Set();
+
+  const pushPrompt = prompt => {
+    if (pushedPrompts.has(prompt)) return;
+    pushedPrompts.add(prompt);
+    promptToShow.set([...get(promptToShow), prompt]);
+  };
+
+  const unsubSpotlight = displayedMessages.subscribe(msgs => {
+    const numAuthors = index(msgs).by('authorId').size;
+    if (numAuthors >= 3) {
+      pushPrompt(FeaturePrompt.SPOTLIGHT);
+    }
+  });
+
+  return () => {
+    unsubSpotlight();
+  };
+};
```

Comment:
Can't this be done in the svelte component script instead of being a separate script?

Reply:
Fixed in e378179a506cf60e94a1a5764354f21ea2fa0b4b